CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Models/GiniCaptureDocument.swift

```diff
@@ -120,5 +120,12 @@ private extension GiniCaptureDocumentBuilder {
 
             self.data = data
         }
+
```

Comment:
[Clarification] @igor-gini could you please check if we still need `public` in `override public func load()`

Reply:
I don't think we need it to be public, removed it there